src/simulators/matrix_product_state/matrix_product_state.hpp

```diff
@@ -645,7 +649,9 @@ void State::apply_measure(const reg_t &qubits,
 }
 
 rvector_t State::measure_probs(const reg_t &qubits) const {
-  return qreg_.probabilities(qubits);
+  rvector_t probvector;
+  qreg_.probabilities_vector(probvector, qubits);
```

Comment:
I thnik is clearer if the `MPS::probabilities_vector` memeber function is renamed to:
```c++
rvector_t probvector = qreg_.get_probabilties_vector(qubits);
```
it makes explicit the intention of the memeber function: returning a new data structure.